crates/worker/Cargo.toml

```diff
@@ -34,7 +34,11 @@ restate-partition-store = { workspace = true }
 restate-rocksdb = { workspace = true }
 restate-serde-util = { workspace = true, features = ["proto"] }
 restate-service-client = { workspace = true }
-restate-service-protocol = { workspace = true, features = [ "codec", "awakeable-id", "message" ] }
+restate-service-protocol = { workspace = true, features = [
+  "codec",
+  "awakeable-id",
+  "message",
+] }
```

Comment:
Can we keep this on a single line please? it makes it easy to re-sort the block.

Reply:
My editor does auto formatting on save, I will change it back and make sure to avoid that in the future